plugins/auth-node/package.json

```diff
@@ -39,6 +39,7 @@
     "@backstage/errors": "workspace:^",
     "@backstage/types": "workspace:^",
     "@types/express": "*",
+    "@types/express-serve-static-core": "^4.17.5",
```

Comment:
Is this one still needed?

Reply:
I don't think so, I removed it and Typescript is not complaining